docs/DeveloperGuide.adoc

```diff
@@ -544,6 +544,58 @@ Use case resumes at step 1.
 +
 Use case resumes at step 1.
 
+[discrete]
+=== Use case: UC09: Browse
+*MSS*
```

Comment:
**MSS** is not showing up as a heading on the Netlify preview (likewise for UC10 and UC11).
```suggestion
*MSS*

```